README.md

```diff
@@ -35,38 +35,40 @@ From your shell or command line:
 
 1. Sign in to the [Azure portal](https://portal.azure.com).
 2. On the top bar, click on your account and under the **Directory** list, choose the Active Directory tenant where you wish to register your application.
-3. Click on **More Services** in the left hand nav, and choose **Azure Active Directory**.
-4. Click on **App registrations** and choose **Add**.
-5. Enter a friendly name for the application, for example "SinglePageApp-DotNet" and select "Web Application and/or Web API" as the Application Type. For the sign-on URL, enter the base URL for the sample, which is by default `https://localhost:44326/`. Click on **Create** to create the application.
-6. While still in the Azure portal, choose your application, click on **Settings** and choose **Properties**.
-7. Find the Application ID value and copy it to the clipboard.
-8. For the App ID URI, enter `https://<your_tenant_name>/SinglePageApp-DotNet`, replacing `<your_tenant_name>` with the name of your Azure AD tenant. 
-9. Grant permissions across your tenant for your application. Go to Settings -> Required Permissions, and click on the **Grant Permissions** button in the top bar. Click **Yes** to confirm.
+3. From the left hand navigation pane, choose **Azure Active Directory**.
+4. Click on **[App registrations](https://go.microsoft.com/fwlink/?linkid=2083908)** and choose **New registration**.
+5. When the Register an application page appears, enter your application's registration information:
+    - Enter a friendly name for the application, for example 'SinglePageApp-DotNet'.
+    - Leave **Supported account types** on **Accounts in this organizational directory only**
+    - For the **Redirect URI (optional)**, enter the base URL for the sample, which is by default `https://localhost:44326/`.
+6. Click on **Register** to create the application.
+7. On the following app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
+8. Select **Expose an API**, and select *Set* For the App ID URI. When prompted, enter `https://<your_tenant_name>/SinglePageApp-DotNet`, replacing `<your_tenant_name>` with the name of your Azure AD tenant.

```

Comment:
I don't think that this sample exposes an API, Navya?

Reply:
Fixed this. Thanks for the catch.